layer/contracts/definitions.py

```diff
@@ -64,6 +64,12 @@ def __init__(
     def __repr__(self) -> str:
         return f"FunctionDefinition({self.asset_type}, {self.asset_name})"
 
+    def _get_source(self) -> str:
```

Comment:
inspect returns an error in case of function code contains variables defined outside of that block so what should be the behavior on model versioning ? Increase the version on each time ?